docs/source/tutorial.rst

```diff
@@ -294,3 +294,77 @@ The stack of DataPipes can then be constructed using their functional forms (rec
 
 In the above example, ``datapipes1`` and ``datapipes2`` represent the exact same stack of ``IterDataPipe``\s. We
 recommend using the functional form of DataPipes.
+
+Working with Cloud Storage Providers
+---------------------------------------------
+
+Accessing AWS S3 with ``fsspec`` DataPipes
```

Comment:
would it be worth having fsspec in requirements.txt? It jsut seems so broadly useful

Reply:
I still support making `fsspec` an optional requirement to maintain the minimum requirements to install `torchdata`. And, even with `fsspec` installed, users still need to install specific lib for each cloud vendor like `s3fs`. So, I feel the extra effort for users are not that significant.